scripts/tests/twister_blackbox/test_verbose.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+"""
+Blackbox tests for twister's command line functions
```

Comment:
We'd need a bit more specific comment here - just mention `--verbose` or something similar.